libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -270,6 +336,11 @@ export class ModalWrapperComponent implements Modal, AfterViewInit, OnInit, OnDe
     //clean up the observer
     this.mutationObserver && this.mutationObserver.disconnect();
     delete this.mutationObserver;
-    this.resizeObserverService && this.resizeObserverService.unobserve(window.document.body);
+    this.resizeObserverService &&
+      this.resizeObserverService.unobserve(this.windowRef.document.body);
```

Comment:
`this.resizeObserverService` is never null or undefined:
```suggestion
    this.resizeObserverService.unobserve(this.windowRef.document.body);
```